.github/workflows/deployment-images.yaml

```diff
@@ -18,6 +18,10 @@ on:
         required: false
         type: string
         default: ''
+      deployment-test-image-name:
```

Comment:
It looks like people are keeping inputs in alphabetical order.

Reply:
fixed. my brain just assumed it already in alphabetical order due to length 🤦